tests/unit_tests/test_tables.py

```diff
@@ -116,6 +121,59 @@ def test_pass_check_time_intervals_stop_after_start():
     assert check_time_intervals_stop_after_start(time_intervals) is None
 
 
+def test_pass_check_column_binary_capability():
+    table = DynamicTable(name="test_table", description="")
+    table.add_column(name="test_col", description="")
+    for x in [1.0, 2.0, 3.0]:
+        table.add_row(test_col=x)
+    assert check_column_binary_capability(table=table) is None
+
+
+def test_pass_array_check_column_binary_capability():
+    table = DynamicTable(name="test_table", description="")
+    table.add_column(name="test_col", description="")
+    for x in [[1.0, 2.0], [2.0, 3.0], [1.0, 2.0]]:
+        table.add_row(test_col=x)
+    assert check_column_binary_capability(table=table) is None
```

Comment:
I think it would be better to test if all of the individual elements of the matrix are 2 values, e.g. all 0s and 1s. This comes up with image masks in `PlaneSegmentation` objects, which are often binary ndarrays and sometimes they mistakenly use floats.

Reply:
You mean add a failing check (where the data is indeed boolean but stored as float, thus returning an `InspectorMessage`), but in array? This particular test is passing because there are 3 unique values among all the elements of the 3x2 array.

Reply:
oh, ok, I think I misunderstood and that you are doing this right. I didn't realize this was passing the check